autotest/gdrivers/mrf.py

```diff
@@ -362,6 +362,27 @@ def test_mrf_lerc_with_huffman():
     gdal.Unlink('/vsimem/out.lrc')
     gdal.Unlink('/vsimem/out.til')
 
+def test_raw_lerc():
+    if 'LERC' not in gdal.GetDriverByName('MRF').GetMetadataItem('DMD_CREATIONOPTIONLIST'):
+        pytest.skip()
+
+    # Defaults to LERC2
+    for opt in 'OPTIONS=V1:1', None:
+        co = ['COMPRESS=LERC']
+        if opt:
+            co.append(opt)
+        gdal.Translate('/vsimem/out.mrf', 'data/byte.tif', format='MRF',
+                        creationOptions = co)
+        ds = gdal.Open('/vsimem/out.lrc')
```

Comment:
@rouault 
This line is the difference, this test opens the data file directly, not the MRF.  Since the data file is a single LERC tile, it will be recognized and opened (without metadata).  This functionality is used in the WMS driver, to read data from services that send back LERC formatted responses.

Reply:
ok, got it